module/userProjects/helper.js

```diff
@@ -1545,13 +1573,13 @@ module.exports = class UserProjectsHelper {
 
                 //Fetch user profile information by calling sunbird's user read api.
 
-                // let userProfile = await userProfileService.profile(userToken, userId);
-                // if ( userProfile.success && 
-                //      userProfile.data &&
-                //      userProfile.data.response
-                // ) {
-                //     createProject.userProfile = userProfile.data.response;
-                // } 
+                let userProfile = await userProfileService.profile(userToken, userId);
```

Comment:
@Prajwal17Tunerlabs use the key in env and based on that call the user service of elevate or sunbird